### DISCUSSION TITLE
Documents are still referring office-ui-fabric-react package

### DISCUSSION THREAD

**Discussion by soumyart** [microsoft/fluentui#17438] [Q&A] [answered]
Documents website (aka.ms/fluentui-react) is still referring office-ui-fabric-react. Any plan to update it with @fluentui/react package and updated props?

![image](https://user-images.githubusercontent.com/20027561/111303959-254f8580-867b-11eb-93a9-0d54237353c9.png)

**Comment by JustSlone** [comment, marked as answer]
Oh, yeah good catch. the "Show Code" still using office-ui-fabric-react. Do you mind filing an issue for this? The fix is probably pretty easy if you are looking to contribute (@ecraig12345  is your goto person for more info on the website).
